redisbench_admin/utils/utils.py

```diff
@@ -383,6 +383,51 @@ def wait_for_conn(conn, retries=20, command="PING", should_be=True, initial_slee
     return result
 
 
+def make_dashboard_callback(
+    return_code, ci_job_name, tf_github_repo, tf_github_branch, tf_github_sha
+):
+    status = "success"
+    if return_code != 0:
+        status = "failed"
+    github_token = os.getenv("GITHUB_TOKEN", None)
+    if github_token is None:
+        logging.error("-- github token is None. Callback skipped --")
+        return
+    workflow_id = os.getenv("CIRCLE_WORKFLOW_ID", None)
```

Comment:
Why do we care about CircleCI here? We can also be called from Github Actions of even from a workstation, no?